doc/yagpdbcc_templates.txt

```diff
@@ -0,0 +1,359 @@
+have string type as keys.
+
+Function                Description
+-----------------------------------------------------------------------------
+sdict "key1" val1... .. Create a new sdict with given key-value pairs.
+
+Method                  Description
+-----------------------------------------------------------------------------
+.Del "key" ............ Delete given key.
+.Get "key" ............ Retrieves given value from key.
+.HasKey "key" ......... Whether this sdict has given key.
+.Set "key" value ...... Set given key to value -- creates new key if it does
+                        not exist already.
+
+                   https://docs.yagpdb.xyz/reference/templates#templates.sdict
+
+
+Tickets~
+                                                            *yagpdbcc-tickets*
+-----------------------------------------------------------------------------
+Ticket functions are limited to one call per custom command for both normal
+and premium guilds.
+
+Function                Description
+-----------------------------------------------------------------------------
+createTicket author topic Create new ticket with given author and topic.
+
+Field                   Description
+-----------------------------------------------------------------------------
+.AuthorID ............. ID of the author of this ticket.
+.AuthorUsernameDiscrim  The discord tag of this ticket's author.
+.ChannelID ............ Channel ID of this ticket.
+.ClosedAt ............. Time that the ticket was closed. Mostly useless.
+.CreatedAt ............ Time this ticket was created.
+.GuildID .............. Guild ID of this ticket.
+.LocalID .............. The ticket's ID.
+.LogsID ............... Log ID of this ticket.
+.Title ................ Title of the ticket.
+
+                           https://docs.yagpdb.xyz/reference/templates#tickets
+
+
+Time~
+                                                               *yagpdbcc-time*
+-----------------------------------------------------------------------------
+Time in general uses Go's time package library, found at https://pkg.go.dev/time.
+
+Field                   Description
+-----------------------------------------------------------------------------
+.DiscordEpoch ......... Gives you Discord Epoch time as time.Time.
+.UnixEpoch ............ Gives you UNIX epoch (1st Jan 1970) as time.Time.
+.TimeHour ............. One hour as time.Duration type.
+.TimeMinute ........... One minute as time.Duration type.
+.TimeSecond ........... One second as time.Duration type.
+
+                              https://docs.yagpdb.xyz/reference/templates#time
+
+User~
+                                                                *yagpdbcc-user*
+-----------------------------------------------------------------------------
+All entries in this section are to be preceded with `.User`, or a valid user
+object.
+
+Special case: a stand-alone `.User` returns the username together with the
```

Comment:
This feels confusingly worded; `.User` does not return the tag; it's simply that `User` implements `fmt.Stringer` and so when user structs are stringified, they display as tags. It follows that when one writes `{{ .User }}`, the `String` method will be called and so the username will be returned.

Perhaps something like "when user structs are stringified, they will display as a combination of the username along with the discriminator" would suffice, but the utility of that is questionable. Removing this sentence altogether might be more appropriate. Speaking from experience, when I first learnt custom command syntax, I read a similar phrase in the documentation and was fairly confused when I wrote `"footer" (sdict "text" .User)` in an embed and received an error, expecting that it would be treated as a string.

Reply:
I agree, I am still weirded out by the phrasing every time I step over it.

Removing it entirely seems to be the best option to me in this case, as any attempt to explain it properly might only confuse users more.
What do you think?

Reply:
Addressed in 1665061